src/Catty/ViewController/CatrobatTableViewController/CatrobatTableViewController.m

```diff
@@ -346,6 +346,13 @@ - (void)configureImageCell:(UITableViewCell <CatrobatImageCell>*)cell atIndexPat
                 dispatch_async(queue, ^{
                     [self.tableView endUpdates];
                 });
+            } else {
+                cell.iconImageView.image = [UIImage imageNamed:@"catrobat"];
```

Comment:
Please return the _catrobat_ image directly at the `loadPreviewImageAndCache` method to prevent code duplications and ease the testing (please add another test to `CBFileManagerTests`).